src/lib/hooks/useNotificationHandler.ts

```diff
@@ -161,10 +161,10 @@ export function useNotificationsHandler() {
 
     const responseReceivedListener =
       Notifications.addNotificationResponseReceivedListener(e => {
-        if (e.notification.request.identifier === prevIdentifier.current) {
+        if (e.notification.date === prevDate.current) {
           return
         }
-        prevIdentifier.current = e.notification.request.identifier
+        prevDate.current = e.notification.date
```

Comment:
squeezing this in here. talked to devin, we're using `identifier` for another purpose it seems (I think gorush is messing this up, it's supposed to always be unique...) Date is also unique though (it is ridiculously precise...), so we will use that instead.